core/common-util/src/StreamUtil.ts

```
@@ -0,0 +1,148 @@
+      typeof (obj as Transform)._transform === 'function';
+  }
+
+  /**
+   * 安全地处理 stream 错误
+   */
+  static handleStreamError(stream: Stream, errorHandler?: (error: Error) => void): void {
+    if (!this.isStream(stream)) {
+      throw new Error('Invalid stream object');
+    }
+
+    const handler = errorHandler || ((error: Error) => {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)